lib/ansible/modules/crypto/openssl_publickey.py

```diff
@@ -90,9 +90,25 @@
         - success
     type: string
     sample: /etc/ssl/public/ansible.com.pem
+fingerprint:
+    description: The fingerprint of the public key. Fingerprint will be generated for each hashlib.algorithms available.
+    returned:
```

Comment:
`returned:` needs to be a string, we are adding in validation of that soon
Could you please rewrite as 
`returned: changed or success`